services/convert/user.go

```diff
@@ -37,16 +39,23 @@ func ToUsers(ctx context.Context, doer *user_model.User, users []*user_model.Use
 
 // ToUserWithAccessMode convert user_model.User to api.User
 // AccessMode is not none show add some more information
-func ToUserWithAccessMode(ctx context.Context, user *user_model.User, accessMode perm.AccessMode) *api.User {
+func ToUserWithAccessMode(ctx context.Context, user, owner *user_model.User, accessMode perm.AccessMode) *api.User {
```

Comment:
```suggestion
func ToUserWithAccessMode(ctx context.Context, user, doer *user_model.User, accessMode perm.AccessMode) *api.User {
```

Reply:
I think they are not always the same

Reply:
@lunny any more feedback?

Reply:
I mean about the variables name. I think the second user should be the doer and the first one is the owner?